libyear/export_format/json.py

```diff
@@ -1,17 +1,45 @@
-from typing import TextIO
+from typing import TextIO, List
+
+from pydantic import BaseModel
 
 from libyear.export_format.export_format import Item
 
 
+class Dependency(BaseModel):
+    name: str
+    version: str
+    latest_version: str
+    libyear: str
+
+
+class JsonOutput(BaseModel):
+    dependencies: List[Dependency] = []
+    libyears_behind: int = 0
+
+
 class JSONFormatter:
+    sort: bool  # Used to sort with libyear attribute and keep same initial behaviour
+    json_output: JsonOutput
+
     def initialize(self, sort: bool):
-        pass
+        self.sort = sort
+        self.json_output = JsonOutput()
 
     def add_item(self, item: Item):
-        pass
+        self.json_output.dependencies.append(
+            Dependency(
+                name=item.name,
+                version=item.version,
+                latest_version=item.latest_version,
+                libyear=item.libyear
+            )
+        )
 
     def end(self, days: int):
-        pass
+        self.json_output.libyears_behind = days
+
+        if self.sort:
+            self.json_output.dependencies.sort(key= lambda dep : (dep.libyear, dep.name))
```

Comment:
I sort by `libyear` and also `name` to list it properly and also to avoid flaky tests at the same time :)